### PR TITLE
Marketplace Thank You: Move the contact link to the topbar

### LINKED ISSUE #74736: Marketplace Thank You: Move the contact section from footer to the topbar
From: 

CleanShot 2023-03-21 at 17 29 35@2x

To: 

CleanShot 2023-03-21 at 17 30 00@2x

---

If quick, add: 

CleanShot 2023-03-21 at 17 32 06@2x

Otherwise just point to the same place and create a ticket to show this helpdesk.

### PR DESCRIPTION
**Pull request by WBerredo**
Fixes #74736

## Proposed Changes

* Move the `MasterbarStyled` to a Marketplace Checkout-specific folder
* Add the contact section as default on this component
* Add the link to support on the contact section
* Remove the contact section from the footer

CleanShot 2023-03-22 at 15 57 31@2x

## Testing Instructions
* Go to the thank you page with a plugin selected. Ex: `marketplace/thank-you/:site?plugins=:plugin-slug`
* Check if the footer section for contact (24/7....) is removed
* Check if a contact section is shown on the topbar
* Click on the link and check if it opens a new tab the link `https://wordpress.com/support/`
* Open the thank you page with a theme selected and check if the same applies. Ex: `marketplace/thank-you/:site?themes=:theme-slug`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by lupus2k — APPROVED** [review]
It works well, but the 2 bottom cards look like are not centered, and I think that is because the text is left-aligned

image

**Review by epeicher — APPROVED** [review]
I have tested it, and it works as advertised. I would create a follow-up task for the footer alignment as it looks not centered. It also looks more prominent for themes because it only has one card 
CleanShot 2023-03-23 at 10 44 09@2x


I agree with @lupus2k that it seems to be due to the left alignment, and I have also found some padding in the paragraph that increase the perceived misalignment
https://github.com/Automattic/wp-calypso/blob/249f7b8a2d77422f6f2252da7ce882241c66473b/client/components/thank-you/index.tsx#L54-L55

In summary, I think this task can progress, and we can work on the footer as part of other tasks

**Comment by WBerredo** [comment]
Thanks for the investigation @epeicher, this can be tackled as part of #74739
